tests/gtests/graph/unit/backend/dnnl/test_conv.cpp

```diff
@@ -2548,6 +2548,9 @@ TEST(test_conv_execute, ConvBiasAddEltwise) {
             // We noticed mish test has slight accuracy issue on GPU or AArch64
             // CPU or SNB.
             dst = eltwise_dst_ts.as_vec_type<float>();
+#ifdef DNNL_AARCH64
+            ASSERT_NEAR(dst[i], param.ref_dst[i], 1e-6);
```

Comment:
Could you share which test case is failing?
From the output you shared, it seems to be leakyRelu.
If that is the case, could you just change line 2450 
- alpha to `.25f` for this test case so that it can be represented exactly in float
- and change reference values with `{-4.0f, 2.5f, 3.0f, 0.5f} * .25f` = `{-1, 0.625, 0.75, 0.125}`

Let me know if that fixes the issue.

Reply:
Hi Mourad, I am not really sure if I should enrol for Hogwarts, or I have discovered a bug, because while trying out your suggestion I have figured out that the test passes without the contribution, on the same machine and instance.... I know this sounds like sorcery but I have now spent some significant amount of time trying to reproduce this. I have tried with different compiles, commits and build configurations but it passed. As a result, we decided that we will closely monitor the behaviour of this specific test and will report any updates that we have. Has this behaviour been encountered before? Do you have any idea what could cause such issues? In case any issues may appear related to this please @ me. 

Thanks!